### DISCUSSION TITLE
F3000 Inverter Support

### DISCUSSION THREAD

**Discussion by bengrins** [nathanmarlor/foxess_modbus#601] [Q&A]
Is it possible to connect an F3000 inverter? I've hooked up a WaveShare RS232/485 TO ETH in the same way as my H1, so assume it's just a case of getting the settings correct.

**Comment by canton7** [comment]
See #220, and iirc some discussion on the Facebook group as well.

To the best of my knowledge it doesn't talk modbus, and noone's been able to get a response out of it.

**Reply by bengrins** [reply]
Thank you! I'll dig around more.
